docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2591,7 +2591,7 @@ The `grpc_client` block configures the gRPC client used to communicate between t
 [max_send_msg_size: <int> | default = 104857600]
 
 # (advanced) Use compression when sending messages. Supported values are:
-# 'gzip', 'snappy' and '' (disable compression)
+# 'gzip', 'snappy', 's2', 's2-snappy' and '' (disable compression)
```

Comment:
```suggestion
# 'gzip', 'snappy', 's2', 's2-snappy', and '' (disable compression).
```

Reply:
We use the Oxford (or serial) comma in docs.

Reply:
This file is auto-generated from command line flags defined in a library Mimir depends on. Ergo we can't change it (the file).